src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -61,14 +68,23 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName, @Reque
         try {
             final EventType eventType = eventTypeRepository.findByName(eventTypeName);
```

Comment:
I don't want to mess up the stats with metrics for event types that doesn't exist. But you are right that is an important call regarding performance and should be measured.